README.md

```diff
@@ -7,14 +7,37 @@
 
 A minimal clone of [TagBot](https://github.com/JuliaRegistries/TagBot) for registries hosted on GitLab.
 
-Creates tags, releases, and changelogs when Julia packages are registered.
+Automatically merges merge requests that were made to register new versions of package, creates tags, and sets the corresponding release notes when Julia packages are registered in a private registry repository hosted on GitLab.
+The merge requests originate from a private Registrator deployment (see [Registrator.jl](https://github.com/JuliaRegistries/Registrator.jl)) but this can be used on any GitLab repository where you wan the same actions to happen independently from where the MRs originate.
```

Comment:
```suggestion
The merge requests originate from a private Registrator deployment (see [Registrator.jl](https://github.com/JuliaRegistries/Registrator.jl)) but this can be used on any GitLab repository where you want the same actions to happen independently from where the MRs originate.
```